doc/multiprocess.md

```diff
@@ -23,9 +23,9 @@ src/bitcoin-node -regtest -printtoconsole -debug=ipc
 BITCOIND=bitcoin-node test/functional/test_runner.py
 ```
 
-The configure script will pick up settings and library locations from the depends directory, so there is no need to pass `--enable-multiprocess` as a separate flag when using the depends system (it's controlled by the `MULTIPROCESS=1` option).
```

Comment:
A few lines above here. This
```
cd <BITCOIN_SOURCE_DIRECTORY>
make -C depends NO_QT=1 MULTIPROCESS=1
CONFIG_SITE=$PWD/depends/x86_64-pc-linux-gnu/share/config.site ./configure
make
src/bitcoin-node -regtest -printtoconsole -debug=ipc
BITCOIND=bitcoin-node test/functional/test_runner.py
```

should be replaced with:
```bash
cd <BITCOIN_SOURCE_DIRECTORY>
make -C depends NO_QT=1 MULTIPROCESS=1
cmake -B build --toolchain=depends/x86_64-pc-linux-gnu/toolchain.cmake
cmake --build build
build/src/bitcoin-node -regtest -printtoconsole -debug=ipc
BITCOIND=build/src/bitcoin-node build/test/functional/test_runner.py
```

Although, it also looks like that is currently broken:
```bash
BITCOIND=build/src/bitcoin-node build/test/functional/test_runner.py
Temporary test directory at /tmp/test_runner_₿_🏃_20240913_111920
2024-09-13T11:19:20.313000Z TestFramework (INFO): PRNG seed is: 4821438991961925939
2024-09-13T11:19:20.315000Z TestFramework (INFO): Initializing test directory /tmp/test_runner_₿_🏃_20240913_111920/cache
2024-09-13T11:19:20.316000Z TestFramework (ERROR): Unexpected exception caught during testing
Traceback (most recent call last):
  File "/root/ci_scratch/test/functional/test_framework/test_framework.py", line 131, in main
    self.setup()
  File "/root/ci_scratch/test/functional/test_framework/test_framework.py", line 301, in setup
    self.setup_chain()
  File "/root/ci_scratch/test/functional/test_framework/test_framework.py", line 392, in setup_chain
    self._initialize_chain()
  File "/root/ci_scratch/test/functional/test_framework/test_framework.py", line 853, in _initialize_chain
    self.start_node(CACHE_NODE_ID)
  File "/root/ci_scratch/test/functional/test_framework/test_framework.py", line 553, in start_node
    node.start(*args, **kwargs)
  File "/root/ci_scratch/test/functional/test_framework/test_node.py", line 256, in start
    self.process = subprocess.Popen(self.args + extra_args, env=subp_env, stdout=stdout, stderr=stderr, cwd=cwd, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.12/subprocess.py", line 1955, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'build/src/bitcoin-node'
2024-09-13T11:19:20.367000Z TestFramework (INFO): Stopping nodes
2024-09-13T11:19:20.367000Z TestFramework (WARNING): Not cleaning up dir /tmp/test_runner_₿_🏃_20240913_111920/cache
2024-09-13T11:19:20.367000Z TestFramework (ERROR): Test failed. Test logging available at /tmp/test_runner_₿_🏃_20240913_111920/cache/test_framework.log
2024-09-13T11:19:20.367000Z TestFramework (ERROR): 
2024-09-13T11:19:20.368000Z TestFramework (ERROR): Hint: Call /root/ci_scratch/test/functional/combine_logs.py '/tmp/test_runner_₿_🏃_20240913_111920/cache' to consolidate all logs
2024-09-13T11:19:20.368000Z TestFramework (ERROR): 
2024-09-13T11:19:20.368000Z TestFramework (ERROR): If this failure happened unexpectedly or intermittently, please file a bug and provide a link or upload of the combined log.
2024-09-13T11:19:20.368000Z TestFramework (ERROR): https://github.com/bitcoin/bitcoin/issues
2024-09-13T11:19:20.368000Z TestFramework (ERROR): 
Traceback (most recent call last):
  File "/root/ci_scratch/build/test/functional/test_runner.py", line 950, in <module>
    main()
  File "/root/ci_scratch/build/test/functional/test_runner.py", line 571, in main
    run_tests(
  File "/root/ci_scratch/build/test/functional/test_runner.py", line 624, in run_tests
    subprocess.check_output([sys.executable, tests_dir + 'create_cache.py'] + flags + ["--tmpdir=%s/cache" % tmpdir])
  File "/usr/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/usr/bin/python3', '/root/ci_scratch/build/test/functional/create_cache.py', '--cachedir=/root/ci_scratch/build/test/cache', '--configfile=/root/ci_scratch/build/test/functional/../config.ini', '--tmpdir=/tmp/test_runner_₿_🏃_20240913_111920/cache']' returned non-zero exit status 1.
```

Reply:
thanks for the feedback! 

I updated this to 
```
cd <BITCOIN_SOURCE_DIRECTORY>
make -C depends NO_QT=1 MULTIPROCESS=1
cmake -B build --toolchain=depends/x86_64-pc-linux-gnu/toolchain.cmake
cmake --build build
build/src/bitcoind -regtest -printtoconsole -debug=ipc
BITCOIND=$(pwd)/build/src/bitcoind build/test/functional/test_runner.py
```

Which does start running the tests please let me know if this looks ok

Reply:
Wouldn't we want to keep `bitcoin-node` (part of multiprocess) rather than change to `bitcoind`? Am I missing something?

Thoughts @ryanofsky ?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30875#discussion_r1759786194

Thanks for the ping! Yes, this should say `bitcoin-node` not `bitcoind` to test the multiprocess binary. Either command line will work, but `bitcoin-node` will make the test more meaningful, especially with changes from #10102 merged, because `bitcoind` will run tests with the node and wallet in the same process, while `bitcoin-node` will spawn separate `bitcoin-wallet` processes

Reply:
I went ahead and changed it back from `bitcoind` to `bitcoin-node` in [dc2e223](https://github.com/bitcoin/bitcoin/pull/30875/commits/dc2e2231036ec09fac4529d54881457fa9a7dc51)